descriptors/ModuleDescriptor-template.json

```diff
@@ -259,6 +247,7 @@
           ],
           "permissionsDesired": [
             "invoices.acquisitions-units-assignments.assign",
+            "invoices.acquisitions-units-assignments.manage",
```

Comment:
seems there is no need to add "invoices.acquisitions-units-assignments.manage" permission here for Orchid and for Poppy branches, as far as I can see it is not used during invoice creation.
Please, add "organizations.organizations.collection.get" permission as modulePermission above (as it was added in the PR https://github.com/folio-org/mod-data-import/pull/310). Looks like it makes sense to have it in the Orchid branch as well.

Reply:
fixed!